jerry-core/ecma/builtin-objects/ecma-builtin-helpers-date.c

```diff
@@ -301,41 +301,6 @@ ecma_date_week_day (ecma_number_t time) /**< time value */
   return (week_day < 0) ? (7 + week_day) : week_day;
 } /* ecma_date_week_day */
 
-/**
- * Helper function to get local time zone adjustment.
- *
- * See also:
- *          ECMA-262 v5, 15.9.1.7
- *
- * @return local time zone adjustment
- */
-static inline ecma_number_t JERRY_ATTR_ALWAYS_INLINE
-ecma_date_local_tza (jerry_time_zone_t *tz) /**< time zone information */
-{
-  return tz->offset * -ECMA_DATE_MS_PER_MINUTE;
-} /* ecma_date_local_tza */
-
-/**
- * Helper function to get the daylight saving time adjustment.
- *
- * See also:
- *          ECMA-262 v5, 15.9.1.8
- *
- * @return daylight saving time adjustment
- */
-static inline ecma_number_t JERRY_ATTR_ALWAYS_INLINE
-ecma_date_daylight_saving_ta (jerry_time_zone_t *tz, /**< time zone information */
-                              ecma_number_t time) /**< time value */
-{
-  JERRY_ASSERT (!ecma_number_is_nan (time));
-
-  /*
-   * TODO: Fix daylight saving calculation.
-   *       https://github.com/jerryscript-project/jerryscript/issues/1661
-   */
-  return tz->daylight_saving_time * ECMA_DATE_MS_PER_HOUR;
-} /* ecma_date_daylight_saving_ta */
-
 /**
  * Helper function to get local time from UTC.
```

Comment:
The comment of this function (`ecma_date_local_time_zone`) became (was?) misleading. Especially when considered together with the other changed function, `ecma_date_utc`. That latter one is documented as "get utc from local time" which is aligned to the code, but this current one has an inverse doc ("get local time from UTC") although it does not implement the inverse operation. Could be updated in this patch.

Reply:
I noticed that, but I wasn't sure if you guys would want it fixed. I renamed it to `ecma_date_local_time_zone_adjustment` and updated the docs as well.